tests/test_deploy_github_pages.py

```diff
@@ -25,7 +26,8 @@ def test_remote_branch_creation(setup_test_env):
         "--target-branch", target_branch,
         "--source-branch", source_branch,
         "--module-path", "../test_package"]
-    CliRunner().invoke(cli.main, args_list)
+    result = CliRunner().invoke(cli.main, args_list)
```

Comment:
The invoke function has a catch_excecptions option, see https://click.palletsprojects.com/en/7.x/api/#click.testing.CliRunner.invoke 
You can than ask the return value of invoke, for the exception with the exception property